sway/config.c

```diff
@@ -549,43 +549,38 @@ static bool load_include_config(const char *path, const char *parent_dir,
 	return true;
 }
 
-bool load_include_configs(const char *path, struct sway_config *config,
+void load_include_configs(const char *path, struct sway_config *config,
 		struct swaynag_instance *swaynag) {
 	char *wd = getcwd(NULL, 0);
 	char *parent_path = strdup(config->current_config_path);
 	const char *parent_dir = dirname(parent_path);
 
 	if (chdir(parent_dir) < 0) {
-		free(parent_path);
-		free(wd);
-		return false;
+		goto out_chdir;
 	}
 
 	wordexp_t p;
 
 	if (wordexp(path, &p, 0) != 0) {
```

Comment:
Invert this, have through wordfree in here and remove the `out_wordexp` label

Reply:
Are you sure? Don't think we can call `wordfree` until later.

Reply:
Move everything that uses the wordexp in there (it's only 6 lines)

Reply:
The blank line before the if can also be removed